Gemfile

```diff
@@ -20,32 +20,30 @@ gem 'jbuilder', '~> 2.7.0'
 gem 'nokogiri', '>= 1.8.2'
 gem 'puma', '~> 5.5.1'
 gem 'rack', '~> 2.2.3'
-gem 'rails', '~> 5.2'
+gem 'rails'
 gem 'rake'
-# gem 'sprockets', '~> 3.7.0', '>= 3.7.2'
 
 gem 'pg'
-gem 'sequel'
+gem 'sequel', '~> 5.51.0'
 gem 'sequel-pg_advisory_locking'
 gem 'sequel-postgres-schemata', require: false
 gem 'sequel-rails'
 
 gem 'activesupport'
 gem 'base32-crockford'
-gem 'bcrypt', '~> 3.1.2'
+gem 'bcrypt'
 gem 'gli', require: false
 gem 'listen'
-gem 'slosilo', '~> 2.2'
+gem 'slosilo', github: 'cyberark/slosilo', branch: 'adjust-to-ruby-3'
```

Comment:
We should get the Slosilo changes released prior to release. I commented on the WIP PR for Slosilo.

Reply:
Released, update slosilo gem version to 3.0.0